### PR TITLE
Nazanin/Mga disclosure requirements

### PR DESCRIPTION
**Pull request by nazaninreihani**

### CODE REVIEW

File: src/templates/app/_layout/footer.jsx [hunk 1]
```diff
@@ -181,6 +184,9 @@ const Footer = () => (
                                     <a className='lga-icon' href='https://www.authorisation.mga.org.mt/verification.aspx?lang=EN&company=a5fd1edc-d072-4c26-b0cd-ab3fa0f0cc40&details=1' target='_blank' rel='noopener noreferrer'>
                                         <img className='responsive' src={it.url_for('images/pages/footer/mga-logo-footer.svg')} />
                                     </a>
+                                    <div className='age-restriction'>
```
Comment (kellybinary) [inline src/templates/app/_layout/footer.jsx]: i think you can move this icon outside of `lga-gamstop-icon-container`

Comment (nazaninreihani) [reply src/templates/app/_layout/footer.jsx]: yes I just found out what it looks like for EUs :-D

Comment (kellybinary) [reply src/templates/app/_layout/footer.jsx]: hehe, i wonder why it's named `lga` instead of `mga` tho.. 🤔

Comment (nazaninreihani) [reply src/templates/app/_layout/footer.jsx]: I think we can change that, it was supposed to be mga probably

Comment (kellybinary) [reply src/templates/app/_layout/footer.jsx]: that'd be great, thank you :)

**Pull request merged**